packages/bot/src/transformers/types.ts

```diff
@@ -1786,6 +1788,17 @@ export interface Nameplate {
   palette: string
 }
 
+export interface UserPrimaryGuild {
+  /** The id of the primary guild */
+  identityGuildId?: bigint
+  /** Thether the user is displaying the primary guild's server tag. This can be `null` if the system clears the identity, e.g. because the server no longer supports tags. */
```

Comment:
```suggestion
  /** Whether the user is displaying the primary guild's server tag. This can be undefined if the system clears the identity, e.g. because the server no longer supports tags. */
```

Technically after the transformer run, this key is not set, so it's "not defined", so I'm debating this

Alternatively we could keep the wording like Discord, it's just wrong since we intentionally skip null values